src/Features/Skylighting.cpp

```diff
@@ -34,9 +34,9 @@ void Skylighting::DrawSettings()
 			"More physically correct, but may impact the intended visual of certain weathers.");
 
 	ImGui::SliderFloat("Diffuse Min Visibility", &settings.MinDiffuseVisibility, 0, 1, "%.2f");
-	ImGui::SliderFloat("Diffuse Brightness", &settings.DiffuseBrightness, 0.3, 3, "%.1f");
+	ImGui::SliderFloat("Diffuse Power", &settings.DiffusePower, 0.3, 3, "%.1f");
```

Comment:
Can you fix these to be `0.3f`, my VS doesn't like the f missing

Reply:
done

Reply:
done